quipucords/fingerprinter/__init__.py

```diff
@@ -226,6 +251,7 @@ def _process_source(report_id, source):
 
         if fingerprint is not None:
             fingerprint['report_id'] = report_id
+            fingerprint[SOURCES_KEY] = [int(source_id)]
```

Comment:
A nit and fake, but could throw an error if source_id wasn't an int.

Reply:
yep.